src/pages/wrap-token-transactions/show.tsx

```
@@ -1,31 +1,56 @@
-import { useMemo } from 'react';
+import { useCallback, useMemo } from 'react';
 import { Button, Stack, TextField } from '@mui/material';
-import { useShow } from '@refinedev/core';
+import { useShow, useUpdate } from '@refinedev/core';
```

ESLint

[error] 'useUpdate' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)